graphql/appointment/fields.ts

```diff
@@ -151,12 +153,14 @@ export class ExtendedFieldsLectureResolver {
                 return appointment.title || 'Kein Titel';
         }
     }
-    @FieldResolver((returns) => String)
-    @Authorized(Role.USER)
-    zoomMeetingId(@Root() appointment: Appointment): string {
```

Comment:
There is already a default field resolver for the field (annotated with authorization checks for PARTICIPANT and OWNER). I'm not sure if every USER is supposed to know the MeetingId? Or doesn't it matter?

Reply:
That default should be totally fine, I guess.